benchmarks/tpcc/src/main/java/com/google/cloud/pgadapter/tpcc/entities/Customer.java

```diff
@@ -0,0 +1,246 @@
+package com.google.cloud.pgadapter.tpcc.entities;
+
+import jakarta.persistence.Basic;
+import jakarta.persistence.Column;
+import jakarta.persistence.EmbeddedId;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinColumns;
+import jakarta.persistence.Lob;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "customer")
+public class Customer {
+
+  @EmbeddedId
+  private CustomerId id;
+
+  @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
I would lean towards making this FetchType.EAGER. The decision between LAZY and EAGER here depends on how often you get a customer and then call getDistrict(). If the latter happens (almost) in all cases that you get a customer, then you should mark this as EAGER.